### PR TITLE
[Semi-Modular] Cybersun Overhaul

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
A companion piece to my lavaland syndicate overhaul (and the second part in my ghostrole overhaul schitck I'm working on), this is a complete overhaul of Cybersun, not limited to aesthetics.
This PR changes:
Cybersun operatives now each have their own room. No bolts built in, but a crafty crewmember can rig one up with a few signallers and a button assembly.
The captain's bedroom has been completely redesigned, and one of the Cybersun hardsuits has been moved there.
A clothesmate has been added, in case everyone looking the same got on your nerves. Stylize!
Medbay is no longer a complete joke, and has the outlines for a cloning and chemistry setup. It's still up to the crew to actually make these, however.
A little library area has been squeezed in, along with a dedicated module for RND and hydroponics.
The cremator has been replaced with an organ harvester pending "review" from the Animal Rights Consortium.
A dedicated cryo room and kitchen have been added! Rejoice!
The RND crate has been moved onto the ship.
Speaking of, the ship looks a lot more war-torn now.
Cargo pod's loot has changed up. In place of an esword, the captain (or whatever crew member asks them for the code first) can get their hands on a chameleon skillchip, an inducer, an industrial RCD, and the usual chameleon gear. Intact still is the codespeak book and other cybersun hardsuit, along with materials.
The agent ID card has multipled into 5, so your space explorer can actually share the access they get from space ruins with the rest of the crew.
Screenshot:
image

## Why It's Good For The Game
Beautification and more sandbox potential for the cybersun personnel who don't just roleplay as Coldsteel the Hedgehog, without taking away from RP opportunities.

## Changelog
:cl:
tweak: Cybersun has been completely overhauled. Pop into CSBC-12!
tweak: Cybersun no longer spawns EVERY shift.
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
ok bro ill think about it

**Comment by Zenitheevee** [comment]
> tweak: Cybersun no longer spawns EVERY shift.

I think this... May not be the best tweak when considering you're also advocating towards RP opportunities...

**Comment by Aphast** [comment]
needs some policy rework thrown in as thats the main problem with cybersun currently, not a lack of cryo and kitchen

**Comment by unit0016** [comment]
I'm just doing the mapping, not the guy to do policy. Also have no idea why it's no longer spawning every shift as normal but it's a FEATURE I GUESS! As for policy, see my separate syndiecomms branch. I was working towards making them more isolated and less involved with traitors per the new admeme agreement earlier today.

**Comment by Oscari-pog** [comment]
Why remove it from spawning every shift if people play it

**Comment by unit0016** [comment]
Realized this is gonna throw a runtime because I didn't switch the space ruin blacklist to the new file, but I'm too sleepy to bother fixing it ATM. I'll get around to it when I wake up.

File: modular_skyrat/modules/mapping/code/datums/ruins/space.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/datum/map_template/ruin/space/forgottenship
+	name = "CSBC-12"
+	id = "forgottenship"
+	description = "Recently upgraded with popular designs, CSBC-12 has managed to maintain it's incredible track record of not veering off into NT controlled space.. 0% of the time."
```
Comment (KathrinBailey) [inline modular_skyrat/modules/mapping/code/datums/ruins/space.dm]: I don't think anyone ever sees this description, but for the sake of it...
Could you just remove the hint of it being 'recently upgraded'? The whole vibe of it is that it's old and it drifted, like Charlie Station, just hasn't been drifting for hundreds of years.

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
Some quick things:
• If there isn't already, a toolbelt would be nice. Chuck an experimental welder (no welder fuel tank), hand drill and jaws down for QoL if there isn't any already.
• Flesh out the kitchen. Add a knife, rolling pin, tray, all of the stuff a kitchen usually has. Keep it compact. Throw in some donk pockets and a bit of basic food.
• Add one sink in hydroponics and the kitchen.

These are my hard requirements, I do like the changes so far.
I need to check a few more things first, so I might request more changes later. I just feel like you've been waiting on this for a bit long.

**Comment by Zenitheevee** [comment]
> 
> 
> I'm just doing the mapping, not the guy to do policy. Also have no idea why it's no longer spawning every shift as normal but it's a FEATURE I GUESS! As for policy, see my separate syndiecomms branch. I was working towards making them more isolated and less involved with traitors per the new admeme agreement earlier today.

wait
You mean its not popping up every round on PRIVATE testing?

Because, I'm gonna be honest, Even on my work on the Charlie station PR i may be dropping for either a modular rehaul alltogether, or in favour for a new, different replacement, I found out that it has to do with server settings, not in-code changes.

File: modular_skyrat/modules/mapping/code/datums/ruins/space.dm (hunk 1 above)
Comment (unit0016) [reply modular_skyrat/modules/mapping/code/datums/ruins/space.dm]: Admins see the description when jumping to it through the debug menu, usually used for flavor text and background lore. I'll change it up, sure! Donk pockets are already there though, I used the same random spawner most maps and base cybersun uses.

**Comment by unit0016** [comment]
> > I'm just doing the mapping, not the guy to do policy. Also have no idea why it's no longer spawning every shift as normal but it's a FEATURE I GUESS! As for policy, see my separate syndiecomms branch. I was working towards making them more isolated and less involved with traitors per the new admeme agreement earlier today.
> 
> wait
> You mean its not popping up every round on PRIVATE testing?
> 
> Because, I'm gonna be honest, Even on my work on the Charlie station PR i may be dropping for either a modular rehaul alltogether, or in favour for a new, different replacement, I found out that it has to do with server settings, not in-code changes.

yeah, it's not. I have no idea why it's not, but I welcome it, personally.

**Comment by Gandalf2k15** [comment]
Seems good. I'll accept.

**Pull request merged**